packages/cheqd/src/anoncreds/utils/identifiers.ts

```diff
@@ -44,7 +45,7 @@ export function parseCheqdDid(didUrl: string): ParsedCheqdDid | null {
     const parts: ParsedCheqdDid = {
       did: `did:cheqd:${sections[1]}:${sections[2]}`,
       method: 'cheqd',
-      network: sections[1],
+      network: sections[1] as `${CheqdNetwork}`,
```

Comment:
Do we need to do string interpolation for the cast?

Reply:
It's not an enum value, but rather the string value. I don't think it really matters, but i also use it as otherwise it will complain if you don't do `CheqdNetowrk.Testnet` (which we don't want in the public api, just `testnet` is easier)